app/uk/gov/hmrc/plasticpackagingtax/registration/forms/LiabilityWeight.scala

```diff
@@ -32,7 +32,7 @@ object LiabilityWeight {
   val weightOutOfRangeError = "liabilityWeight.outOfRange.error"
 
   private val isWithinRange: Option[Long] => Boolean = weight =>
-    minTotalKg to maxTotalKg contains weight.getOrElse(minTotalKg)
```

Comment:
This change was it improve performance BTW.   Before the change I'd noticed that 'LiabilityWeightTest' was "stuttering" for a few seconds when you ran `sbt test`